dts/arm/nxp/nxp_rt.dtsi

```diff
@@ -800,19 +800,27 @@
 			#pinmux-cells = <2>;
 			reg = <0x40384000 0x4000>;
 			clocks = <&ccm IMX_CCM_SAI1_CLK 0x7C 2>;
-			pre-div = <0>;
-			podf = <63>;
-			pll-clocks = <&anatop 0x70 0xC000 0>,
-					<&anatop 0x70 0x7F 32>,
-					<&anatop 0x70 0x18 1>,
-					<&anatop 0x80 0x3FFFFFFF 77>,
-					<&anatop 0x90 0x3FFFFFFF 100>;
+			// Audio PLL Output Frequency is determined by:
```

Comment:
@ioannisg Should `checkpatch` be catching these comments? [Zephyr coding guidelines](https://docs.zephyrproject.org/latest/contribute/guidelines.html#coding-style) suggest these should be C89 comments:
> Use C89-style single line comments, /*  */. The C99-style single line comment, //, is not allowed.